src/book/03-guides/04-general/04-documentation.mdx

```diff
@@ -0,0 +1,78 @@
+---
+section: Guides
+chapter: General
+title: Documentation
+description: How to properly document code.
+slug: /guides/general/documentation
+---
+
+## Conventions
+
+We use the javadoc documentation style for comments that are able to be formatted with doxygen. Each variable, function and class declaration should have a comment detailing the information about it in the `.hpp` file.
+
+### Directives
+
+This is a list of directives that we use.
+
+- `@brief` A brief description of the following code. Everything should at least have this tag.
+- `@detail` A complete and detailed description of the code.
+- `@param` Details a paramter that is passed to a function.
```

Comment:
`@param` usually names the parameter as the first term and then a description. This should tell us that. It should also clarify formatting details. For example, is there a colon or hyphen between the parameter name and description? Your example indicates that there isn't, but if this is a reference, it should tell us that IMO.
```suggestion
- `@param` Details a parameter that is passed to a function.
```